tests/Microsoft.Identity.Test.Unit.net45/SeleniumWrapper.cs

```diff
@@ -0,0 +1,100 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.IO;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+
+namespace Microsoft.Identity.Test.Unit
+{
+    public class SeleniumWrapper : IDisposable
+    {
+        public SeleniumWrapper(bool headlessMode = true)
+        {
+            ChromeOptions chromeOptions = new ChromeOptions();
+
+            if (headlessMode)
```

Comment:
perhaps this can be settable from outside of the application with a config file or an env variable. This way, it will be easy for us to turn the ui on so that we can watch what happens on the build machines without having to change code on the branch.

Reply:
nit: You can also chose to skip exposing headlessMode as this is currently only needed headlessly (except when creating/debugging a test)?

---
In reply to: [240713488](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/718#discussion_r240713488) [](ancestors = 240713488)